webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    entry: './src/index.tsx',
+    output: {
+        path: path.join(__dirname, '/dist'),
+        filename: 'bundle.js'
+    },
+    devServer: {
+        hot: true,
+        compress: true,
+        open: true,
+        port: 3000,
+        historyApiFallback: true
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

```suggestion
        filename: 'bundle.js',
    },
    devServer: {
        hot: true,
        compress: true,
        open: true,
        port: 3000,
        historyApiFallback: true,
```